apps/catalogue/components/landing/CardPrimary.vue

```diff
@@ -22,7 +24,19 @@ withDefaults(defineProps<Props>(), {
         <BaseIcon :name="image" :width="55" />
       </span>
       <div class="relative">
-        <NuxtLink :to="link" :target="openLinkInNewTab ? '_blank' : undefined">
+        <a
+          class="font-display md:text-heading-5xl text-heading-5xl text-title-contrast-pop px-3"
+          :href="link"
+          v-if="renderAsAnchorElem"
+          :target="openLinkInNewTab ? '_blank' : undefined"
+        >
+          {{ title }}
+        </a>
+        <NuxtLink
+          v-else
+          :to="link"
+          :target="openLinkInNewTab ? '_blank' : undefined"
+        >
           <h1
             class="font-display md:text-heading-5xl text-heading-5xl text-title-contrast-pop px-3"
           >
```

Comment:
This does not need to be a heading and pages should not have more than one h1 element. These classes should be moved to the link instead.

Reply:
Removed heading element. The heading styles are part of the link element